src/shift-reader.js

```diff
@@ -58,6 +58,9 @@ const binaryOps = ["+", "-", "*", "/", "%","<<", ">>", ">>>", "&", "|", "^",
 
 const unaryOps = ["++", "--", "~", "!", "delete", "void", "typeof", "yield", "throw", "new"];
 
+// Any -> Boolean
+const isNotFalse = R.compose(R.not, R.curry(R.equals)(false));
```

Comment:
Fair enough :)
I just was trying to emulate the existing style as much as I could. I can change that to a simple function instead.